scripts/superdesk-search/services/TagService.js

```diff
@@ -23,6 +23,13 @@ export function TagService($location, desks, userList, metadata, search) {
 
     var cvs = search.cvs;
 
+    function tag(label, value) {
+        return {
+            label: label,
+            value: value || label
+        };
+    }
+
     function initSelectedParameters (parameters) {
         tags.selectedParameters = [];
```

Comment:
it's not only there, but all over the service. might give es6 a go, but is there any advantage to use a function there? just wondering..